internal_transfers/actions/src/simulate/tenderly.ts

```diff
@@ -107,11 +107,12 @@ export function parseTenderlySimulation(
 ): SimulationData {
   const { transaction: tx } = simulation;
   if (tx != undefined) {
+    const balanceDiff = tx.transaction_info.balance_diff ?? [];
     return {
       blockNumber: tx.transaction_info.block_number,
       logs: tx.transaction_info.logs.map((t: { raw: EventLog }) => t.raw) || [],
       ethDelta: new Map(
-        tx.transaction_info.balance_diff.map((t) => [
+        balanceDiff.map((t) => [
```

Comment:
This was not previously picked up by our e2e tests because they are skipped. Turns out this `balance_diff` field can be null (i.e. when the transaction does not cause any change to ETH balance of the contract).